util/build.gradle.kts

```diff
@@ -54,5 +54,12 @@ kotlin {
                 })
             }
         }
+        val commonTest by getting {
+            dependencies {
+                implementation(kotlin("test"))
+                implementation(libs.coroutines.test)
+                implementation(libs.okio.test)
```

Comment:
```suggestion
```

This one can go. We don't have any Okio dependencies, so nothing to test either.